src/Microsoft.Android.Runtime.NativeAOT/Android.Runtime.NativeAOT/JavaInteropRuntime.cs

```diff
@@ -13,7 +13,7 @@ static int JNI_OnLoad (IntPtr vm, IntPtr reserved)
 	{
 		try {
 			AndroidLog.Print (AndroidLogLevel.Info, "JavaInteropRuntime", "JNI_OnLoad()");
-			LogcatTextWriter.Init ();
+			AndroidCryptoNative_InitLibraryOnLoad(vm, reserved);
```

Comment:
We will also need to wire up the native code initialization here, ie. what CoreCLR does:
```cpp
	OSBridge::initialize_on_runtime_init (env, runtimeClass);
	GCBridge::initialize_on_runtime_init (env, runtimeClass);
```